velox/dwio/dwrf/test/TestColumnStatisticsBase.h

```diff
@@ -647,26 +739,40 @@ TEST(StatisticsBuilder, basicMissingStats) {
   EXPECT_TRUE(stats->hasNull().value());
 }
 
-TEST(StatisticsBuilder, basicHasNull) {
+void testBasicHasNull(
+    ColumnStatisticsWrapper columnStatisticsWrapper,
+    DwrfFormat format) {
   enum class State { kTrue = 0, kFalse, kMissing };
-  auto test = [](State to, State from, State expected) {
+  auto test = [&columnStatisticsWrapper, &format](
+                  State to, State from, State expected) {
     StatisticsBuilder target{options};
     if (to == State::kTrue) {
       target.setHasNull();
     } else if (to == State::kMissing) {
       // merge against unknown
-      proto::ColumnStatistics proto;
-      target.merge(*buildColumnStatisticsFromProto(proto, context));
+      target.merge(
+          *buildColumnStatisticsFromProto(columnStatisticsWrapper, context));
     }
 
-    proto::ColumnStatistics proto;
-    if (from == State::kFalse) {
-      proto.set_hasnull(false);
-    } else if (from == State::kTrue) {
-      proto.set_hasnull(true);
+    if (format == DwrfFormat::kDwrf) {
+      proto::ColumnStatistics proto;
+      if (from == State::kFalse) {
+        proto.set_hasnull(false);
+      } else if (from == State::kTrue) {
+        proto.set_hasnull(true);
+      }
+      target.merge(*buildColumnStatisticsFromProto(
+          ColumnStatisticsWrapper(&proto), context));
```

Comment:
the Linter didn't run on this file, could you change this to

target.merge(
  *buildColumnStatisticsFromProto(columnStatisticsWrapper, context));

Reply:
I've refactored this part of the code:
```
auto columnStatistics =
    google::protobuf::Arena::CreateMessage<proto::ColumnStatistics>(
       arena_.get());
if (from == State::kFalse) {
   columnStatistics->set_hasnull(false);
 } else if (from == State::kTrue) {
   columnStatistics->set_hasnull(true);
}
target.merge(*buildColumnStatisticsFromProto(
      ColumnStatisticsWrapper(columnStatistics), context()));
```
Not sure if this will work in Linter. Please ping me if you need me to modify it.

Reply:
@wypb it's the lines

target.merge(*buildColumnStatisticsFromProto(
      ColumnStatisticsWrapper(columnStatistics), context()));

that are triggering the Linter to fail, could you split these into two lines as follows

`
target.merge(
      *buildColumnStatisticsFromProto(ColumnStatisticsWrapper(columnStatistics), context()));
`